src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/SubmitToSaboModal.tsx

```diff
@@ -114,6 +114,28 @@ const SubmitToSaboModal = ({ open, setOpen, reimbursementRequest }: SubmitToSabo
           <DetailDisplay label="Expense Type" content={`${expenseType.code} - ${expenseType.name}`} />
         </Grid>
       </Grid>
+      <Grid container spacing={1} sx={{ marginTop: 2 }}>
+        <Grid item xs={4}>
+          <DetailDisplay label="Treasurer" content="" />
+        </Grid>
+        <Grid item xs={8}>
+          <Stack>
+            <Typography sx={{ fontWeight: 'normal', display: 'inline' }}>Brody Pearlman</Typography>
```

Comment:
These props (fontWeight and display) don't actually do anything, could you remove them?